specification/ws/src/main/java/org/kaazing/specification/ws/internal/Functions.java

```diff
@@ -34,6 +34,29 @@
     private static final int MAX_ACCEPTABLE_HEADER_LENGTH = 200;
 
     @Function
+    public static String loginBase64Encoder(String login) {
```

Comment:
This is a general method that converts a string into a base 64 encoded string, it's not specific to login, so I would recommend making in a more general to reflect that. How about encodeAsBase64 or base64Encode?